dist/mask.js

```diff
@@ -424,7 +425,13 @@ angular.module('ui.mask', [])
                                 iElement.val(valMasked);
                                 
                                 scope.$apply(function() {
-                                    controller.$setViewValue(valUnmasked); // $setViewValue should be run in angular context, otherwise the changes will be invisible to angular and user code.
+                                    // $setViewValue/$viewValue should be run in angular context, otherwise the changes will be invisible to angular and user code.
+                                    if (linkOptions.silentEvents.indexOf(eventType) >= 0) {
```

Comment:
I think simplifying this `linkOptions.silentEvents.indexOf(eventType) !== -1` feels a little more idiomatic.

Also, Array.indexOf is ie9+ (I'm assuming we don't actually support below 9 so this is probably fine). Any thoughts @PowerKiKi?